dotcom-rendering/src/web/components/Callout/RadioSelect.tsx

```diff
@@ -25,6 +27,7 @@ export const RadioSelect = ({
 	>
 		<FieldLabel formField={formField} />
 		<RadioGroup
+			error={validationErrors?.[formField.id]}
 			name={formField.name}
```

Comment:
The other way of handling the optional here is to use `?` before we try to access the value of the key.

Reply:
nice!